atproto/auth/oauth/types.go

```diff
@@ -407,3 +413,21 @@ type TokenResponse struct {
 	// Refresh token, for doing additional token requests to the auth server.
 	RefreshToken string `json:"refresh_token"`
 }
+
+// The fields which are included in a token revocation request. These HTTP POST bodies are form-encoded, so use URL encoding syntax, not JSON.
```

Comment:
we haven't been particularly consistent about it, but we could toss in a link to RFC7009 here